src/libaktualizr/uptane/tuf.cc

```diff
@@ -341,7 +341,7 @@ void Uptane::BaseMeta::init(const Json::Value &json) {
   try {
     expiry_ = TimeStamp(json["signed"]["expires"].asString());
   } catch (const TimeStamp::InvalidTimeStamp &exc) {
-    throw Uptane::InvalidMetadata("", "", "Invalid timestamp");
+    throw Uptane::InvalidMetadata("", "", exc.what());
```

Comment:
Why change this one but not the many other similar cases in this file? To me it looks like ideally we should print both the existing message as well as the `what()`, but that should really be a separate PR.

Reply:
> Why change this one but not the many other similar cases in this file?
Because I rant into the case when this throw happens and needed more details on it.
I'll roll back this change.